app/org/sagebionetworks/bridge/services/UserDataDownloadViaSqsService.java

```diff
@@ -52,14 +52,14 @@ public final void setSqsClient(AmazonSQSClient sqsClient) {
     public void requestUserData(@Nonnull StudyIdentifier studyIdentifier, @Nonnull User user,
             @Nonnull DateRange dateRange) throws JsonProcessingException {
         String studyId = studyIdentifier.getIdentifier();
-        String username = user.getUsername();
+        String email = user.getEmail();
         String startDateStr = dateRange.getStartDate().toString();
         String endDateStr = dateRange.getEndDate().toString();
 
         // construct message as string-string map
         Map<String, String> requestMap = new HashMap<>();
         requestMap.put(REQUEST_KEY_STUDY_ID, studyId);
-        requestMap.put(REQUEST_KEY_USERNAME, username);
+        requestMap.put(REQUEST_KEY_USERNAME, email);
```

Comment:
Are there any consequences to this change? Production users have the same value for username and email, so it should appear no different after deployment.